CHANGELOG.md

```diff
@@ -4,17 +4,37 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.20 - 2023-10-03
+
+### Added
+- Documentation for experimental option `--pass-args` [#3229](https://github.com/earthly/earthly/pull/3229).
+- Clarification about `RUN --ssh` no being related to GIT [3eb60e87](https://github.com/earthly/earthly/commit/3eb60e87bbbee965a7c0f5d7b0f89cc2f65a7d02).
+- Support for `mode` in mount cache [#3279](https://github.com/earthly/earthly/pull/3279).
```

Comment:
we generally link to issues rather than PRs -- not all users will understand the changes to our code, but instead would get more from seeing a higher-level description of the bug that was fixed.
```suggestion
- Support for `mode` in mount cache [#3278](https://github.com/earthly/earthly/issues/3278).
```